avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -15,6 +15,8 @@ distanceTimeSteps = 0:1
 thresholdResult = 1.e-5
 # set minimum time step difference for computing approach velocity > 2s for discarding spurious effects
 minVelTimeStep = 2.
+# use the maximum rangeTimeResType or the mean rangeTimeResType (max or mean)
+maxOrMean = max
```

Comment:
this is only applied for the mean or max values of the TT diagram not the range time with radar field of view - so either this should be moved to the tt section of the configuration settings or also implemented for the range time diagram function: extractFrontAndMeanValuesRadar